edi_account_invoice_import_ubl/data/edi_exchange_type.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
```

Comment:
@simahawk I think that it makes sense to add everything together (outgoing and incoming UBL invoices) in one module, I just split it because of what you mentioned: I followed your template in sales and because the "base" modules are already split account_invoice_import_ubl (incoming) and account_invoice_ubl (outoing). I think this "merge" could be part of a migration because it needs to be done in the sales module anyway, and at least now the structure is consistent.

Regarding the installation of default exchange types, it is the same, I followed your example on sales, but I have no problem to switch to the pre-init hooks style if you provide me another example.

Thanks for having a look!

Reply:
ping @simahawk